lte/gateway/python/integ_tests/s1aptests/test_multi_enb_complete_reset.py

```diff
@@ -0,0 +1,117 @@
+"""
+Copyright (c) 2016-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+"""
+import time
+import unittest
+import s1ap_types
+import s1ap_wrapper
+
+
+class TestMultiEnbMultiUEAttachDetach(unittest.TestCase):
+
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_attach_detach_multienb_multiue(self):
+        """ Multi Enb Multi UE attach detach """
+        req = s1ap_types.multiEnbConfigReq_t()
+        # Num of Enbs (Max Number of ENBS is 5)
+        num_of_enbs = 5
+        plmn_length = 6
+        # column is a enb parameter,  row is a number of enbs
+        """            Cell Id,   Tac, EnbType, PLMN Id """
+        enb_list = list([[1,       1,     1,    "001010"],
+                         [2,       1,     1,    "001010"],
+                         [3,       1,     1,    "001010"],
+                         [4,       1,     1,    "001010"],
+                         [5,       1,     1,    "001010"]])
+        # Maximum 5 Enbs can be configured
+        req.numOfEnbs = num_of_enbs
+
+        # ENB Parameter column index initialization
+        cellid_col_idx = 0
+        tac_col_idx = 1
+        enbtype_col_idx = 2
+        plmnid_col_idx = 3
+
+        for idx1 in range(num_of_enbs):
```

Comment:
The procedure from line 45-60 can be converted to a library function, which takes the `enb_list` as parameter and configures `req.multiEnbCfgParam`. This will be reused in all multi eNB tests.